src/components/elements/query-image.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 import styled, { css } from 'styled-components'
 import { GatsbyImage, getImage } from 'gatsby-plugin-image'
-import type { ImageDataLike } from 'gatsby-plugin-image'
+import type { ImageDataLike, IGatsbyImageData } from 'gatsby-plugin-image'
 
 type QueryImageProps = {
     alt: string
     className?: string
```

Comment:
Should not this className, height and width be always string ?

Reply:
Sorry I don't quite understand your question. Yep it is now currently set to only be string.

Reply:
I meant as it says with the '?', it can be undefined as well. Do we have a chance this className can be undefined in this component?, If not, we can remove this otherwise we can keep this.

Reply:
Yes the `?` is present as these are optional props passed to QueryImage. They may not be used at times.